webrtc.html

```diff
@@ -4306,6 +4307,11 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <h2>Methods</h2>
             <dl data-link-for="RTCCertificate" data-dfn-for="RTCCertificate"
             class="methods">
+              <dt><dfn><code>getSupportedAlgorithms</code></dfn></dt>
+              <dd>
+                <p>Returns a sequence of supported and recommended certificate algorithms.
```

Comment:
I'm not sure "recommended" is the right term. It would be reasonable for an implementation to return an algorithm that uses a SHA-1 hash, if it's supported, but it wouldn't be recommended. So I'd suggest saying something like "Returns a representative set of supported certificate algorithms."

It would also help to have a note section that clarifies further. Here's a suggestion:

> For example, the `"RSASSA-PKCS1-v1_5"` algorithm dictionary, `RsaHashedKeyGenParams`, contains fields for the modulus length, public exponent, and hash algorithm. Implementations are likely to support a wide range of modulus lengths and exponents, but a finite number of hash algorithms. So in this case, it would be reasonable for the implementation to return one `AlgorithmIdentifier` for each supported hash algorithm that can be used with RSA, using default/recommended values for `modulusLength` and `publicExponent` (like 1024 and 65537, respectively).